libs/users/src/users.model.ts

```diff
@@ -1,36 +1,26 @@
 import { ObjectID } from 'bson';
 import { Exclude, classToClass } from 'class-transformer';
 import { ApiProperty } from '@nestjs/swagger';
+import { AccessTokenResponse } from './auth/auth.model';
 
-export const socialAuthType = ['apple', 'google'] as const;
-export type SocialAuthType = typeof socialAuthType[number];
-
-export class UuidRegTokenPair {
-  [uuid: string]: string;
+export enum SocialAuthTypeEnum {
+  Apple = 'apple',
+  Google = 'google',
 }
 
 export class User {
-  @ApiProperty()
+  @ApiProperty({ type: String })
   public _id: ObjectID;
   @Exclude()
   public socialAuthId?: string;
-  @ApiProperty()
-  public socialAuthType?: SocialAuthType;
-  @ApiProperty()
+  public socialAuthType?: SocialAuthTypeEnum;
   public createdAt?: Date;
-  @ApiProperty()
   public updatedAt?: Date;
-  @ApiProperty()
   public lastLoginAt?: Date;
-  @ApiProperty()
   public isAdmin?: boolean;
-  @ApiProperty()
   public isAnonymous?: boolean;
-  @ApiProperty()
-  public uuidRegTokenPair?: UuidRegTokenPair;
-  @ApiProperty()
+  public uuidRegTokenPair?: string;
```

Comment:
this one is not string, it s this type

Reply:
Yes, I know but it was [key: string]: string and it's not allowed in swagger 4, if we use UuidRegTokenPair is we get empty Object in swagger model